Chains/multishine.py

```diff
@@ -22,7 +22,7 @@ def step(self, gamestate, smashbot_state, opponent_state):
                 return
             if smashbot_state.action_frame == 2:
                 self.interruptible = False
-                controller.empty_input()
+                controller.tilt_analog(Button.BUTTON_MAIN, int(smashbot_state.facing), .5) #hold forward for advancing JC shine
```

Comment:
Presumably there's times where we might want to multishine in place without moving right? Or maybe even move backwards? I'd rather this be an argument that the caller has to specify, as opposed to always moving forward.

Reply:
Personally, I'm not sure there's any reason we would want to multishine in place or while retreating, but I do agree that it makes sense we should add an argument which allows you to specify forward, backward, or in place.

Reply:
I'd be concerned about moving off the stage, in partcular